### PR TITLE
Setup A/B test for QuickAdd component

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
Add A/B test for QuickAdd component with 50/50 weight

Screenshot 2024-06-05 at 14 31 26

Screenshot 2024-06-05 at 14 32 17

## Justify why they are needed
New A/B test setup

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4343?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4343** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4343?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4342** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4342?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gustaveen and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/src/components/QuickAdd/QuickAddOfferContainer.tsx [hunk 1]
```diff
@@ -56,7 +61,12 @@ export function QuickAddOfferContainer(props: Props) {
 
   const homeInsuranceInCart =
     props.cart.entries.find((entry) => HOME_INSURANCES.includes(entry.product.name)) ?? null
-  if (!homeInsuranceInCart && Features.enabled('CROSS_SELL_CARD_V2')) {
+
+  const experimentCookie = getCookie(EXPERIMENT_COOKIE_NAME)
+  const experimentVariant = getExperimentVariant(experimentCookie)
+  const showDefaultVariant = !experimentVariant || experimentVariant.id === 0
```
[the remark this answers is not in the document]
Comment (alebedev) [reply apps/store/src/components/QuickAdd/QuickAddOfferContainer.tsx:67]: Does it assume we only have single running experiment?  If so, could you comment that it'll break with multiple experiments?

**Review by alebedev — APPROVED** [review]

File: apps/store/src/components/QuickAdd/QuickAddOfferContainer.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (gustaveen) [reply apps/store/src/components/QuickAdd/QuickAddOfferContainer.tsx:67]: > Does it assume we only have single running experiment? If so, could you comment that it'll break with multiple experiments?

It does. Let's add a comment if we merge without supporting multiple tests

**Comment by gustaveen** [comment]
### Merge activity

* **Jun 10, 5:28 AM EDT**: @gustaveen started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4343).

**Pull request merged**